scripts/sync_claude_distribution.py

```
@@ -792,6 +820,36 @@ def _render_marketplace(repo_root: Path) -> dict[str, object]:
+    manifest = load_json(
+        repo_root
+        / "plugins"
+        / "creative-writing-skills"
+        / ".codex-plugin"
+        / "plugin.json"
+    )
+    name = manifest["name"]
+    if _ZCODE_PLUGIN_NAME_RE.fullmatch(name) is None:
+        raise UnsupportedTransformError(
+            f"plugin name {name!r} is not a valid ZCode plugin name"
+        )
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)